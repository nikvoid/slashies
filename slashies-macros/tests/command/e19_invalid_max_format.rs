use slashies_macros::Command;

/// An invalid command
#[derive(Command)]
#[name = "BadCommand"]
struct BadCommand {
    /// Field with badly formatted max attribute
    #[max]
    bad_field: i64,

    /// Ok field
    #[max = 0]
    good_field: i64,
}

fn main() {}
