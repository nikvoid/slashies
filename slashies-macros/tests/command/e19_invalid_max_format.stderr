error: Invalid "miax" attribute. Attribute must be of the form #[max = value]
 --> tests/command/e19_invalid_max_format.rs:8:5
  |
8 |     #[max]
  |     ^^^^^^
